src/PanelTraits/AutoSet.php

```diff
@@ -14,9 +14,23 @@
      */
     public function setFromDb()
     {
+        $this->setDoctrineTypesMapping();
         $this->getDbColumnTypes();
 
-        array_map(function ($field) {
+        $crudTypes = null;
+        $crudFakeFields = null;
+
+        if(!is_null($this->model->crudTypes)) {
+            $crudTypes = $this->model->crudTypes;
+        }
+
+        if(!is_null($this->model->crudFakeFields)) {
+            $crudFakeFields = $this->model->crudFakeFields;
+        }
+
+        //dd($crudTypes);
```

Comment:
Yeah, needs to be removed was there for my testing purposes. I rather spit it out web page then view in a file.